asset/package.json

```diff
@@ -1,10 +1,12 @@
 {
   "name": "file-assets",
-  "version": "0.2.1",
+  "version": "0.3.0",
   "description": "A set of processors for working with files",
   "dependencies": {
     "@terascope/chunked-file-reader": "^0.1.2",
     "@terascope/job-components": "^0.4.1",
```

Comment:
We should update `job-components` to the latest, `^0.14.4`, based on the usage I don't believe it breaks anything.

Reply:
updated